020-Connect/010-Postgres/001-direct-access.mdx

```diff
@@ -227,6 +227,23 @@ Tables created through the wire protocol will be missing the columns necessary t
 
 Please reference to the [special columns](/docs/concepts/data-model#special-columns) section to learn more about these columns.
 
+## Column metadata
+
+Xata uses SQL comments on columns to track some replication-related metadata about certain types of columns. These comments are only ever present on columns created using the Xata UI, and only on certain types of columns.
+
+The full list of column types and their associated metadata comments is as follows:
+
+| Xata Type | Metadata comment                    | Notes                          |
+| --------- | ----------------------------------- | ------------------------------ |
+| email     | `{"xata.type":"email"}`             |                                |
+| vector    | `{"xata.search.dimension":n}`       | dimension `n` is set in the UI |
+| string    | `{"xata.type":"string"}`            |                                |
+| text      | `{"xata.type":"text"}`              |                                |
+| file      | `{"xata.file.dpa":false}`           |                                |
+| link      | `{"xata.link": "referenced_table"}` |                                |
```

Comment:
Small design suggestion. I'd remove the backticks from the code snippets in the table. Because of line breaks / table formatting it's really hard to style these things to look good. I recently changed everything table based to mono font for this reason, so you shouldn't need the extra formatting.

Reply:
Backticks removed in 432734d0d2054342c1225303c8a896de16a69fcc